src/components/skip-link/index.md.njk

```diff
@@ -15,7 +15,9 @@ Use the skip link component to help keyboard-only users skip to the main content
 
 ## When to use this component
 
-All GOV.UK pages must include a skip link in the header.
+All GOV.UK pages must include a skip link, placed on its own at the top of the `<body>` tag, and outside of the `<header>' tag.
+
+Do not put the skip link inside the `<header>` tag. axe will flag it as a failure because axe expects all elements to be inside a landmark.
```

Comment:
Elements or components?

Reply:
I know it is but do you want to specifically mention axe or maybe 'accessibility checking tools'? Just looks a bit odd with the on-brand lower case 'a' and some may not be a familiar with it.

Reply:
@jbuller That's a great point, thanks! Will change.

```suggestion
Do not put the skip link inside the `<header>` tag. Accessibility testing tools that expect all elements to be inside a landmark will flag it as a failure.
```